magma_scripts/MWSieve-x1_18.m

```diff
@@ -56,17 +81,21 @@ function doOurMWSieve(d)
 
     print "flag 1 and flag2 for d = ", d, " are respectively", flag1, "and", flag2;
     
+    // Check the computation actually worked, and if not, return a string
     if not flag1 then
         return "MW computation failed";
     end if;
 
+    // The desired basis
     bas := [J1!(MWtoSet(MW.1)), J1!(MWtoSet(MW.2))];
 
+    // Here we do the MW sieve
     ans := HasPointMWSieve(J1, bas, deg3: testfun := func<p, v | IsOdd(v) and p gt 3>);  // this is true or false
 
     return ans;
 end function;
 
+// Run it for the d we need to check
 for d in OurDsToCheck do
```

Comment:
also print a list of succesfull and failed d's at the end, so that it is easy to see what the result was. Also we should add a log file containing the output of running this.

Reply:
the code changes are done. however adding the log file is a separate issue now: https://github.com/isogeny-primes/quadratic-torsion/issues/5 since this computation is still running